submissions/BogdanGapon/HTML & CSS Practice/sass/base/_base.scss

```diff
@@ -0,0 +1,38 @@
+
+*,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    color: var(--main-color);
+    margin: 0;
+    padding:0;
+}
+
+
+h1,
+h2,
+h3,
+h4,
+ul,
+p {
+    margin: 0;
+    padding: 0;
+}
+
+img {
+    display: block;
+}
+
+li {
+list-style: none;
```

Comment:
Look at indentation, everything should be in the same style. Add indentations inside blocks.